packages/arb-token-bridge-ui/src/hooks/usePendingTransactions.ts

```diff
@@ -0,0 +1,144 @@
+import { Provider } from '@ethersproject/providers'
+import dayjs from 'dayjs'
+import {
+  fetchDeposits,
+  fetchWithdrawals,
+  OutgoingMessageState
+} from 'token-bridge-sdk'
+import {
+  outgoungStateToString,
+  transformDeposits,
+  transformWithdrawals
+} from '../state/app/utils'
+import useSWR from 'swr'
+import { useNetworksAndSigners } from '../hooks/useNetworksAndSigners'
+import { useAppState } from '../state'
+import { useGateways } from './useGateways'
+import { DepositStatus } from '../state/app/state'
+
+const INITIAL_PAGE_SIZE = 10
+
+export const fetchPendingTransactions = async ({
+  address,
+  l1Provider,
+  l2Provider,
+  gatewayAddresses,
+  pageSize = INITIAL_PAGE_SIZE
+}: {
+  address: string
+  l1Provider: Provider
+  l2Provider: Provider
+  gatewayAddresses: string[]
+  pageNumber?: number
+  pageSize?: number
+  searchString?: string
+}) => {
+  console.log('***** START: fetching pending transactions *****')
```

Comment:
Not related to this PR, but do we want `console.log` in production?

Reply:
Yeah we will not be needing it. It's just done for benchmarking right now. Removing.